pkg/network/ebpf/tuple_types_linux.go

```diff
@@ -0,0 +1,26 @@
+// Code generated by cmd/cgo -godefs; DO NOT EDIT.
+// cgo -godefs -- tuple_types.go
+
+package ebpf
+
+type ConnType uint32
+
+const (
+	UDP ConnType = 0x0
+	TCP ConnType = 0x1
+)
+
+type ConnFamily uint32
+
+const (
+	IPv4 ConnFamily = 0x0
+	IPv6 ConnFamily = 0x2
```

Comment:
Should this be 0x1?

Reply:
Nope. These are bitmask values. Take note this file is generated.